apps/emqx/test/emqx_cth_cluster.erl

```diff
@@ -93,6 +93,8 @@
     %% Default: "${ClusterOpts.work_dir}/${nodename}"
     work_dir => file:name(),
 
+    allow_dirty_work_dir => boolean(),
```

Comment:
I think the original intention from @keynslug  for this module was to ensure we don't have dirty work dirs.  :sweat_smile:

Reply:
Yeah. 🫠 I looked through the code (albeit quickly), and my initial impression is that it doesn't really needed. There's a lot of stuff happening during each testcase setup, so I believe it could be simplified by running `emqx_cth_suite:start/2` for _each testcase_ instead. Another, kind of easier approach is to just run `t_uuid_restored_from_file` under its own `emqx_cth_suite` / `emqx_cth_cluster`, so we won't need to do that awkward stopping in `init_per_testcase` and starting back in `end_per_testcase`.

Reply:
You're right. I'll try to rewrite this test case